recipes/gromacs/2019/meta.yaml

```diff
@@ -44,7 +53,7 @@ test:
 extra:
   identifiers:
     - biotools:gromacs
-    - doi:10.5281/zenodo.2424363
-    - doi:10.5281/zenodo.2424486
+    - doi:{{ doi_release }}
```

Comment:
@stain I'm not sure if this is overdoing it.

It complicates downstream tools that just want to read metadata and with this needs to render the file.

Reply:
Agree, I'll change it back. 

A related question here is which DOI - there is one for each release (Zenodo archive), one for documentation, and one for "cite me". Hope all are OK for now, [Citation File Format](https://citation-file-format.github.io/) could be an alternative that is (too) detailed.